### PR TITLE
Divide elastic docs into sections

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

File: frameworks/elastic/docs/configure.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+You can view the deploy plan for the Elasticsearch framework via the service URL: `http://$DCOS_URL/service/dcos-{{cluster-name}}/v1/plans`
+
+# Topology
+
+Each task in the cluster performs one and only one of the following roles: master, data, ingest, coordinator, kibana.
+
+The default placement strategy distributes all instances of the same node task type to different agents. So no two master nodes would run on the same agent, but a data node might get placed onto an agent that is also running a master node.
+
+![agent](https://s3.amazonaws.com/loren-elastic-assets/Default+Specialized+by+Agent+-+Page+1.png "Private Nodes Grouped by Agent")
```
Comment (loren) [inline frameworks/elastic/docs/configure.md]: Probably time to move these pics somewhere more official.

Comment (sascala) [inline frameworks/elastic/docs/configure.md]: indeed

**Comment by loren** [comment]
retest this please

File: frameworks/elastic/docs/configure.md (hunk 1 above)
Comment (sascala) [inline frameworks/elastic/docs/configure.md]: ...and done

**Comment by sascala** [comment]
retest this please

**Comment by sascala** [comment]
retest this please

**Comment by sascala** [comment]
retest this please

**Comment by loren** [comment]
0/4 so far on this documentation-only PR. This time: 
`* What went wrong:
Could not resolve all dependencies for configuration ':cassandra:compileClasspath'.
> Could not download protobuf-java.jar (com.google.protobuf:protobuf-java:2.6.1)
   > Could not get resource 'https://repo1.maven.org/maven2/com/google/protobuf/protobuf-java/2.6.1/protobuf-java-2.6.1.jar'.
      > Connection reset
`

@gabrielhartmann What do you say to just merging this?

**Pull request merged**